src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -888,6 +900,9 @@
   <data name="ERR_FixedMustInit" xml:space="preserve">
     <value>You must provide an initializer in a fixed or using statement declaration</value>
   </data>
+  <data name="ERR_You_must_provide_an_initializer_for_a_readonly_variable" xml:space="preserve">
+    <value>You must provide an initializer for a 'readonly' variable</value>
```

Comment:
I'm following lots of other error messages here (including the one *right above* this one) :)

Perhaps there should just be a pass through the compiler errors to make them consistent?  We did this on TS and it was a very good thing.

Other things i've noticed: Compiler errors are completely inconsistent with their use of punctuation.  Some end with period, some do no.  No way to tell why.